src/components/PopupWithForm.js

```diff
@@ -25,6 +34,10 @@ export default class PopupWithForm extends Popup {
     });
   }
 
+  setSubmit(handleSubmit) {
```

Comment:
Needs correction: this class should not have such method. You should create a separate class for the confirmation `PopupWithConfirmation`